clue.js

```diff
@@ -0,0 +1,181 @@
+// STEP 1 - CREATE OBJECTS FOR ALL THE SUSPECTS, SOMETHING LIKE THIS:
+
+const mrGreen = {
```

Comment:
Your objects look nice and clearly defined with strings being used where necessary 💯